packages/components/src/Select/Option.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useRef } from 'react';
+import type { ListState } from '@react-stately/list';
+import type { Node } from '@react-types/shared';
+import { useOption } from '@react-aria/listbox';
+
+import { Box } from '../Box';
+
+interface OptionProps {
+  item: Node<unknown>;
+  state: ListState<unknown>;
+}
+
+export const Option = ({ item, state }: OptionProps) => {
+  const ref = useRef<HTMLLIElement>(null);
+  const { optionProps, isSelected } = useOption(
+    {
+      key: item.key,
+    },
+    state,
+    ref
+  );
+
+  return (
+    <Box
+      as="li"
+      {...optionProps}
+      ref={ref}
+      p="2px 5px"
+      variant={isSelected ? 'select.option.selected' : 'select.option.default'}
```

Comment:
Should we make use of theme-ui's `__default` here?

Reply:
Niiiice, it works. Will try it on other parts of the Select, too.